src/styles/body.scss

```diff
@@ -0,0 +1,7 @@
+*
+{
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
it's only allowed to apply `box-sizing: border-box;` to `*` selector.
If you need to reset some default styles you can use tags selectors like `body { margin: 0}`